apps/web/src/utils/format-currency.ts

```diff
@@ -20,7 +21,7 @@ export function formatCurrency(
   if (amount === null || amount === undefined) {
     amount = 0
   } else if (typeof amount === 'string') {
-    amount = Number.parseFloat(amount) * 100
+    amount = Math.round(Number.parseFloat(amount) * 100)
```

Comment:
The function was failing for certain numbers. I can't remember if this one of the problem #s, but if for instance `10.13` was submitted, it would get an amount of `10.1299999` instead, and Dinero would say the number was invalid. This fixes that issue.